common/sagas/rates.js

```diff
@@ -0,0 +1,19 @@
+// @flow
+import { put, call } from 'redux-saga/effects';
+import type { Effect } from 'redux-saga/effects';
+import { setRates } from 'actions/rates';
+
+const symbols = ['USD', 'EUR', 'GBP', 'BTC', 'CHF', 'REP'];
+
+function fetchRates(symbols) {
+  return fetch(
+    `https://min-api.cryptocompare.com/data/price?fsym=ETH&tsyms=${symbols.join(
```

Comment:
i dont like it :D Formatted string are strictly better in places where you can get away with it imo.